tests/cypress/e2e/features/ground_truth_jobs.js

```diff
@@ -385,93 +385,116 @@ context('Ground truth jobs', () => {
+                        cy.goCheckFrameNumber(frame);
+                        checkRectangleAndObjectMenu(groundTruthRectangles[index]);
+                    });
+
+                    cy.interactMenu('Open the task');
+                    cy.get('.cvat-task-job-list').within(() => {
+                        cy.contains('a', `Job #${jobID}`).click();
+                    });
+                    groundTruthFrames.forEach((frame) => {
+                        cy.goCheckFrameNumber(frame);
+                        cy.get('.cvat_canvas_shape').should('not.exist');
+                        cy.get('.cvat-objects-sidebar-state-item').should('not.exist');
+                    });
                 });
-            }).as('delayedRequest');
-
-            cy.contains('button', 'Submit').click({ force: true });
-            cy.contains('button', 'Submit').should('be.disabled');
-            cy.wait('@delayedRequest');
-
-            cy.get('.cvat-canvas-container').should('exist').and('be.visible');
-            cy.url().then((url) => {
-                jobID = Number(url.split('/').slice(-1)[0].split('?')[0]);
-            }).should('match', /\/tasks\/\d+\/jobs\/\d+/);
+            });
         });
-
-        it('Check GT annotations can not be shown in standard annotation view', () => {
-            cy.headlessCreateJob({
-                task_id: taskID,
-                frame_count: 4,
-                type: 'ground_truth',
-                frame_selection_method: 'random_uniform',
-                seed: 1,
-            }).then((jobResponse) => {
-                groundTruthJobID = jobResponse.jobID;
-                return cy.headlessCreateObjects(groundTruthFrames.map((frame, index) => {
-                    const gtRect = groundTruthRectangles[index];
-                    return {
-                        objectType: 'shape',
-                        labelName,
-                        type: 'rectangle',
-                        occluded: false,
-                        frame,
-                        points: [gtRect.firstX, gtRect.firstY, gtRect.secondX, gtRect.secondY],
-                    };
-                }), groundTruthJobID);
-            }).then(() => {
-                cy.visit(`/tasks/${taskID}/jobs/${jobID}`);
-                cy.get('.cvat-canvas-container').should('exist');
-
-                cy.changeWorkspace('Review');
-                cy.get('.cvat-objects-sidebar-show-ground-truth').click();
-                cy.get('.cvat-objects-sidebar-show-ground-truth').should(
-                    'have.class', 'cvat-objects-sidebar-show-ground-truth-active',
+        describe.only('GT jobs from videos', () => {
+            const serverFilesVideo = ['videos/video_without_valid_keyframes.mp4'];
+            before(() => {
+                createAndOpenTask(
+                    serverFilesVideo,
+                    { ...defaultValidationParams, frameCount: 3, randomSeed: 42 },
```

Comment:
```suggestion
                    { ...defaultValidationParams, randomSeed: 42 },
```
Frame count is already 3 in defaultValidationParams